packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -3,12 +3,12 @@
         display: flex;
         flex-direction: column;
         @include desktop {
-            overflow: scroll;
+            overflow: auto;
         }
     }
     &__scrollable-content {
         @include desktop {
-            overflow: scroll;
+            overflow: auto;
```

Comment:
An intermittent issue of displaying scrollbar area in the jurisdiction modal while connecting laptop to monitor was encountered earlier by harvinraj. Today i encountered the same issue and fixing it here.